onnxruntime/contrib_ops/cpu/quantization/qlinear_global_average_pool.cc

```diff
@@ -55,6 +55,33 @@ Status ComputeQLinearGlobalAvgPool(
   return Status::OK();
 }
 
+// we would either put the template function in header file or add such declaration in the below to do explicit instantiation.
```

Comment:
just for the history it would be good to capture why we need this in the comment - namely unexplained gcc behavior when --disable-exceptions and --minimal-build are combined on linux resulting this becoming an undefined symbol. doesn't seem to affect other platforms.

and given that I don't think we'd need/want to put it in the header file

Reply:
Comments updated.